src/App.tsx

```diff
@@ -1,17 +1,139 @@
-import React from 'react';
-import './App.css';
+import React, { useState } from 'react';
+import './App.scss';
 
 import users from './api/users';
+import todos from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+
+const preparedTodos: Todo[] = todos.map((todo) => ({
+  ...todo,
+  user: users.find((customer) => (
+    customer.id === todo.userId)) || null,
+}));
 
 const App: React.FC = () => {
+  const [title, setTitle] = useState('');
+  const [name, setName] = useState('');
```

Comment:
better save user id

Reply:
The name is in the select field. The list is not changeable. I need your additional comment.

Reply:
But I implemented it.